README.md

```diff
@@ -62,20 +61,14 @@ OSM runs an Envoy based control plane on Kubernetes, can be configured with SMI
 
 Read more about [OSM's high level goals, design, and architecture](DESIGN.md).
 
-## Getting Started
-
-Below are quick getting started instructions. For a more detailed example usage guide and demo walkthrough, see the [OSM Example Usage Guide](/docs/example/README.md).
+## Install
```

Comment:
We should retain Getting Started section. Getting Started doesn't mean install, could also mean just reading the docs, design etc.

Reply:
Agreed